dashboard/app/views/admin_reports/level_completions.html.haml

```diff
@@ -2,20 +2,31 @@
   = stylesheet_link_tag    'jquery.handsontable.full', media: 'all'
   = javascript_include_tag 'jquery.handsontable.full'
 %h1 Level Completions
-%h2
-  From
-  %input#dateFrom{style: 'position: relative; z-index: 102;'}
-  To
-  %input#dateTo{style: 'position: relative; z-index: 102;'}
-  Filter
-  %input#filter
-%a#dateUpdate{style: 'display: none'} Update
-%h2
-  #samplingMessage{style: 'color: red; display: none'} CAUTION: Data is sampled.
-%h2 #{link_to 'Download CSV', {:format => :csv }}
-#completionTable
+
+%form{method: 'get'}
+  %h2
+    From
+    %input#dateFrom{name: 'start_date', value: params[:start_date], style: 'position: relative; z-index: 102;'}
+    To
+    %input#dateTo{name: 'end_date', value: params[:end_date], style: 'position: relative; z-index: 102;'}
```

Comment:
these can actually be [date inputs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date)